turbo.json

```diff
@@ -27,6 +27,7 @@
         "tsc",
         "eslint-check",
         "prettier-check",
+        "test-check-rate-limit-snapshots",
         "test-sync"
```

Comment:
This'll run them in parallel, which isn't want we want, is it?

Reply:
I was mostly just trying to get the test in at all tbh.

I agree that it should ideally not be run in parallel. I guess we should run it after `test-sync`  when recording (as you suggested below), and before the tests when not recording (no need to run them all if we know it's incorrect).

Am I correct in thinking this is done just by changing `"test-run"` and `"test-run-record"` in the package.json scripts?

Reply:
> Am I correct in thinking this is done just by changing "test-run" and "test-run-record" in the package.json scripts?

Yup. I think my a suggestion below should do that.